source/base/table_handler.cc

```diff
@@ -268,7 +268,7 @@ TableHandler::add_column_to_supercolumn(const std::string &key,
 {
   Assert(columns.count(key), ExcColumnNotExistent(key));
 
-  if (supercolumns.count(superkey) == 0u)
+  if (!supercolumns.count(superkey))
```

Comment:
Here, and below I would also prefer the previous version since `count` returns an `int`.